checkov/common/sca/output.py

```diff
@@ -666,15 +667,17 @@ def get_license_statuses(packages: list[dict[str, Any]]) -> list[_LicenseStatus]
     if not requests_input:
         return []
     try:
-        response = request_wrapper(
-            method="POST",
-            url=f"{bc_integration.api_url}/api/v1/vulnerabilities/packages/get-licenses-violations",
-            headers=bc_integration.get_default_headers("POST"),
-            json={"packages": requests_input},
-            should_call_raise_for_status=True
-        )
-        response_json = response.json()
-        license_statuses: list[_LicenseStatus] = _extract_license_statuses(response_json)
+        license_statuses: list[_LicenseStatus] = []
+        for chunked_request_input in chunk_array(requests_input, 50):
```

Comment:
How can you tell that 50 is the magic number?
it's dependents on the content size, if one package have a longer name there the other?

Reply:
can't really tell TBH. maybe it is better to dig into and find better number.